index.ts

```diff
@@ -42,7 +42,11 @@ export {
   LiveResponse,
   LiveValidationIssue,
   LiveValidatorOptions,
-  ApiOperationIdentifier
+  ApiOperationIdentifier,
+  RequestValidationResult,
+  ResponseValidationResult,
+  ValidateOptions,
+  formatUrlToExpectedFormat
 } from "./lib/validators/liveValidator"
 export { SpecResolver } from "./lib/validators/specResolver"
```

Comment:
also lets update the `package.json` patch version otherwise publishing will fail

Reply:
Updated to 0.15.4